src/modules/request/request.service.ts

```diff
@@ -130,34 +131,43 @@ export class RequestService {
     }
   }
 
-  findManyWithPagination(paginationOptions: IPaginationOptions, feed?: boolean): Promise<RequestEntity[]> {
+  async findManyWithPagination(paginationOptions: IPaginationOptions, user: User): Promise<ListRequestsDto> {
     const { ordering } = paginationOptions;
 
-    if (feed) {
-      return this.requestRepository.find({
-        skip: (paginationOptions.page - 1) * paginationOptions.limit,
-        take: paginationOptions.limit,
-        order: {
-          id: ordering ? ordering : "ASC",
-        },
-        relations: {
-          godFather: true
-        },
-        where: {
-          godFather: IsNull(),
-          status: {
-            name: Object.keys(RequestStatusEnum).find(key => RequestStatusEnum[key] === RequestStatusEnum.analysis)
-          }
-        }
-      });
-    }
+    const res = await this.requestRepository.find({
+      skip: (paginationOptions.page - 1) * paginationOptions.limit,
+      take: paginationOptions.limit,
+      order: {
+        id: ordering ? ordering : "ASC",
+      },
+    });
+
+    const userId = user.id;
+
+    const forMe = res.filter(request => request.user.id === userId);
+    const forOthers = res.filter(request => request.user.id !== userId);
```

Comment:
Nesse caso é pra mostrar apenas as que o usuário autenticado apadrinhou

```suggestion
    const forOthers = res.filter(request => request?.godFather?.id === userId);
```

Reply:
erro meu